src/frontend/components/Icon/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { appData } from '../../data/appData';
+
+const Svg = styled.svg`
+  width: 1rem;
+  height: 1rem;
+  fill: currentColor;
+`;
+
+interface IconProps {
+  name: string;
+  title?: string;
+}
+
+export const Icon = ({ name, title }: IconProps) => {
+  return (
+    <Svg role="img" aria-hidden={!title}>
+      <use xlinkHref={`${appData.static.svg.icons}#${name}`} />
```

Comment:
```suggestion
      <use href={`${appData.static.svg.icons}#${name}`} />
```
`xlink:href` has been deprecated since SVG 2 in favor of `href`. So if you take in account my comment above about svg version, you should update this.
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href

Reply:
I have trouble finding anything resembling a compatibility table for this. There's this [CSS tricks article](https://css-tricks.com/on-xlinkhref-being-deprecated-in-svg/) which notes incompatibility with some versions of iOS and safari.

Since it's a change that does not bring anything new (and I'm pretty sure `xlinkHref` will be supported for the next 20 years anyway, at the very least) I'd be inclined to use `xlinkHref` and not worry about compatibility.

Reply:
Ok 🤷‍♂️. 
You can find easily this information on caniuse.com https://caniuse.com/mdn-svg_elements_use_href or at the bottom of the MDN documentation, there is a also a compatiblity table.

It shows this attribute is well supported  by all major browsers since 2019.

Reply:
Oh sorry, I did miss this compatibility table. It would still break icons for 1% of users (mainly last year's version of Safari)  for no real benefit so I'd keep it all the same